tests/fast-integration/telemetry-test/suite.js

```diff
@@ -313,15 +314,35 @@ describe('Telemetry Operator', function() {
     });
   });
 
-  context('Configurable Tracing', function() {
+  context.only('Configurable Tracing', function() {
```

Comment:
Is it really the `only` test that should be executed?

Reply:
No, leftover. Thanks